### PR TITLE
Translation tutorial 3

### PR DESCRIPTION
**Pull request by Jzow**
### Description

<Please describe the issues fixed by this PR>

Related Issue: <Insert link here if applicable>

### Pre-commit steps

 - [x] Tested on QEMU and real HW Rasperry Pi.
     - Not needed if it is just a README change or similar.
 - [x] Ran `./contributor_setup.sh` followed by `./devtool ready_for_publish`
     - You'll need `Ruby` with `Bundler` and `NPM` installed locally.
     - If no Rust-related files were changed, `./devtool ready_for_publish_no_rust` can be used instead (faster).
     - This step is optional, but much appreciated if done.

### CODE REVIEW

**Comment by Jzow** [comment]
#166

File: 03_hacky_hello_world/README.CN.md [hunk 1]
```diff
@@ -0,0 +1,371 @@
...
+Hello from Rust!
+Kernel panic!
+
+Panic location:
+      File 'src/main.rs', line 126, column 5
+
+Stopping here.
+```
+
+## 与上面不同
```
Comment (andre-richter) [inline 03_hacky_hello_world/README.CN.md:34]: Please don't copy the diff in translations, but reference the auto-generated english version. See: https://github.com/rust-embedded/rust-raspberrypi-OS-tutorials/blob/master/02_runtime_init/README.CN.md#%E7%9B%B8%E6%AF%94%E4%B9%8B%E5%89%8D%E7%9A%84%E5%8F%98%E5%8C%96diff

**Comment by andre-richter** [comment]
Thanks! Do you have somebody for proofreading?

CC @colachg @readlnh

File: 03_hacky_hello_world/README.CN.md [hunk 2]
```diff
@@ -0,0 +1,371 @@
...
+
+- 全面介绍`println!()` 宏以便尽早启用"printf debugging"。
+- 为了保持教程长度合理，打印函数目前 "滥用" 了 QEMU 属性，该属性允许我们在没有正确设置的情况下使用树莓派的`UART`。
+- 使用真正的硬件`UART 在以下教程中逐步启用。
+
+## 值得注意的补充
+
+- `src/console.rs`为控制台命令和通过`console::console()`对内核控制台的全局访问引入了接口`Traits`。
+- `src/bsp/raspberrypi/console.rs` 实现QEMU仿真UART的接口。
+- 紧急处理程序使用新的`println!()`以显示用户错误消息。
```
Comment (colachg) [inline 03_hacky_hello_world/README.CN.md:13]: panic保留如何，我也不知道翻译成什么比较好。panic处理程序

File: 03_hacky_hello_world/README.CN.md [hunk 3]
```diff
@@ -0,0 +1,371 @@
...
+- 有一个新的Makefile目录`make test`，用于自动测试。它在`QEMU`中引导编译后的内核，并检查内核生成的预期输出字符串。
+  - 在本教程中，它检查字符串`Stopping here`，该字符串由`panic!()`在`main.rs`的末尾。
```
Comment (colachg) [inline 03_hacky_hello_world/README.CN.md:15]: 该字符由`main.rs`末尾的`panic!()`抛出。

**Review by colachg — COMMENTED** [review]
Hi @Jzow , I have given some comments. Thanks for your work :)

**Comment by Jzow** [comment]
@colachg Thank you for reviewing, I try to modify

**Comment by Jzow** [comment]
@colachg Hi, I have tried to modify it. Can you review it again? If ok, I will continue to translate the following chapters

**Comment by colachg** [comment]
> @colachg Hi, I have tried to modify it. Can you review it again? If ok, I will continue to translate the following chapters

Of course, can you please resolve all the comments that I left above if you have modified it or even you don't want to modify it?

**Comment by Jzow** [comment]
> can you please resolve all the comments

please check, I have modified it and resolve all

**Review by colachg — APPROVED** [review]
/lgtm

File: 03_hacky_hello_world/README.CN.md (hunk 1 above)
Comment (andre-richter) [reply 03_hacky_hello_world/README.CN.md:34]: This hasn’t been done it seems. Please change as suggested.

**Review by andre-richter — CHANGES_REQUESTED** [review]
Please see m comment regarding the diff section that was falsely marked as resolved.

**Comment by Jzow** [comment]
Please check [the english version](README.md#diff-to-previous), which is kept up-to-date. 

Don't have to translate this sentence into Chinese? My understanding is that it can be linked to the English version of diff, which can be seen clearly by users. This sentence should be translated

If necessary, I will make another modification

@colachg What do you think?

**Comment by andre-richter** [comment]
Okay for me to translate this sentence 👍

**Comment by andre-richter** [comment]
Please open one PR per chapter. That makes reviewing easier for others.

It seems that for tutorial 3, you still have the English sentence for
> Please check [the english version](README.md#diff-to-previous), which is kept up-to-date.

In tutorial 4, it is translated already.

**Comment by andre-richter** [comment]
Thanks a lot!

**Pull request merged**